index.html

```diff
@@ -4554,6 +4554,16 @@ <h2 id="examples-tzigla">Tzigla</h2>
 
     <h2 id="changelog">Change Log</h2>
 
+    <b class="header" id="1.6.2">1.6.2</b> &mdash; <small><i>Apr. 16, 2025</i></small>
+    &mdash; <a href="https://github.com/jashkenas/backbone/compare/1.6.1...1.6.2">Diff</a>
+    &mdash; <a href="https://cdn.statically.io/gh/jashkenas/backbone/1.6.2/index.html">Docs</a>
+    <br />
+    <ul style="margin-top: 5px;">
+      <li>
+        Upgrade underscore dependency to fix <a href="https://nvd.nist.gov/vuln/detail/CVE-2021-23358">CVE-2021-23358</a>
+      </li>
+    </ul>
+
```

Comment:
Side note: please do not add change log entries, change the Backbone version number or update generated files such as the annotated source. We do those things when we cut a release.